docs/theme/package.json

```diff
@@ -3,6 +3,9 @@
   "main": "js/theme.js",
   "version": "1.0.0",
   "private": true,
+  "resolutions" : {
+    "**/elliptic": "6.5.3"
```

Comment:
```suggestion
    "**/elliptic": "^6.5.3"
```

Reply:
Doesn't `^6.5.3` mean semver-compatible with `6.5.3`? If so, that would include `6.5.2` as well. We probably need something like
```suggestion
    "**/elliptic": "^6.5.3 >=6.5.3"
```
Does that look right?

Reply:
I don’t think so, `^` also adds a lower bound. See https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

Reply:
Yeah, you're right. I stopped reading too early. Apologies.

Reply:
@cocreature -- added the resolutions entry on package.json then reran yarn install so no manual edit of lock file

Reply:
i'll add the range rather than specific version to both